.github/actions/baseline-test/action.yml

```diff
@@ -0,0 +1,222 @@
+runs:
+  using: composite
+  steps:
+      - name: Clone latest hubble code
+        uses: actions/checkout@v2
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        with:
+          repository: jina-ai/hubble
+          token: ${{ secrets.JINA_DEV_BOT }}
+          ref: refs/heads/master
+          path: hubble
+          lfs: true
+
+      - name: mk logs folder
+        run: mkdir logs
+        shell: bash
+      
+      - name: Start containers
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        run: | 
+          mkdir -p /tmp/jina-uploaded /tmp/jina-hubble-temp
+          sudo chown -R $USER:$USER /tmp/jina-uploaded /tmp/jina-hubble-temp
+          docker-compose up -d
+        shell: bash
+        working-directory: hubble
+        
+      - name: setup knative
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        uses: mapleeit/actions-setup-knative@main
+      - run: minikube tunnel --profile knative &> /dev/null &
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        shell: bash
+
+      - name: Use Node.js ${{ inputs.node-version }}
+        uses: actions/setup-node@v2
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        with:
+          node-version: ${{ inputs.node-version }}
+
+      - run: yarn install --frozen-lockfile
+        working-directory: hubble
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        shell: bash
+      - run: npm run build --if-present
+        working-directory: hubble
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        shell: bash
+      - run: npm run start:prod > ../logs/server.log &
+        working-directory: hubble
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        shell: bash
+
+      - name: Use Python ${{ inputs.python-version }}
+        uses: actions/setup-python@v2
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        with:
+          python-version: ${{ inputs.python-version }}
+
+      - name: Clone latest jina code
+        uses: actions/checkout@v2
+        env:
+            NODE_ENV: ${{ inputs.node_env }}
+            JINA_HUBBLE_REGISTRY: ${{ inputs.jina_hubble_registry }}
+        with:
+          repository: jina-ai/jina
+          token: ${{ secrets.JINA_DEV_BOT }}
+          ref: refs/heads/master
+          path: jina
```

Comment:
This also. should be `__actions__jina__`. Otherwise if the original repo already has `jina` folder under the root. This will cause unexpected consequence.